src/main/java/com/endava/java2023/DTO/OrdersDTO.java

```diff
@@ -0,0 +1,29 @@
+package com.endava.java2023.DTO;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import lombok.Setter;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+
+import java.util.Date;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+
+public class OrdersDTO {
+
+    private Integer eventId;
+    private Date timestamp;
+    private Integer ticketCategoryId;
+    private int numberOfTickets;
+    private double totalPrice;
+
+    public OrdersDTO(Integer eventId, Date timestamp, int numberOfTickets, double totalPrice) {
```

Comment:
Why did you write this constructor with arguments as is generated by @AllArgsConstructor ? 
Or you really need to generate a OrdersDTO  without a ticketCategoryId ?